File: ClienteServidor/ProyectoClienteServidor/Cliente/src/Cliente.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.PrintStream;
import java.net.Socket;

public class Cliente {
    public static void main(String[] args) {
        Socket cliente;
        PrintStream p;
        //BufferedReader b;
        {
            try {
                cliente = new Socket("192.168.2.21", 1234);

                p = new PrintStream(cliente.getOutputStream());
                p.println("hola");

            } catch (IOException e) {
                throw new RuntimeException(e);
            }
        }
    }
}

File: UltimoTrabajo/app/src/main/java/com/example/trabajofinalpsp/objetos/Cliente.kt
package com.example.trabajofinalpsp.objetos

import java.io.PrintWriter
import java.net.Socket

class Cliente() : Thread() {

    private lateinit var client: Socket
    private lateinit var out: PrintWriter

    override fun run() {
        super.run()
        println("cualquier")
        //Hago conexion con servidor y envio las cordenadas/pulsacion del boton
        //Servidor y puerto los cojo de los Textview del activity_main2.xml
        client = Socket("192.168.56.1", 1234)
        out = PrintWriter(client.getOutputStream(), true)
        //Envio coordenadas/pulsacion boton derecho o izquierdo --> Depende de que se pulse
        out.println("cord")
        out.close()
        client.close()
    }

}